participants/anatlavitzkovitz.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Anat Lavi-Itzkovitz",
+    "linkedin": "Anat lavi-itzkovitz",
```

Comment:
This is not your username on LinkedIn. This looks like it: https://www.linkedin.com/in/anat-lavi-itzkovitz-88500243/

Reply:
ok I changed my linkedin URL to anat-lavi-itzkovitz and updated the json accordingly